### DISCUSSION TITLE
Deep potential application issue: out of memory

### DISCUSSION THREAD

**Discussion by shuaijiang-ustc** [deepmodeling/deepmd-kit#277] [Q&A]
Hi, DPMD developers!

Recently I got the pb file representing deep potential and tried to apply it with the help of ASE module. The chemical systems for testing compose of multiple frames, so I use deep potential with a 'for cycle'. However, even though at the begining, dozens of GB memory was free, at the end, it always showed that the memory was running out. Besides, nothing is printed out except the following message: 
...
 tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
So I am wondering how to get ride of the memory problem when using deep potential.
Thank you very much!

Shuai

**Comment by njzjz** [comment]
Please post the full logs of the error.
